tools/scoring/score/parse_config.py

```diff
@@ -177,3 +178,57 @@ def retrieve_reporting_translations(
       }
 
     return translations
+
+  @staticmethod
+  def aggregate_results_nondbo(
+      # TODO: create Dimension type to replace generic
+      *,
+      dimensions: List[Callable[[Dict], Dict]],
+      translations: Dict[str, List[Tuple[str, Any]]]) -> Dict[str, Result]:
+    """
+      Wrapper which outputs a dictionary of results by invoking
+      each specified `Dimension` with the `translations` argument
```

Comment:
what is a dimension? the `Result` class is a dimension?

Reply:
A dimension is a metric which is being evaluated for the proposed file (i.e. the core functionality). I'd planned to have them separate but maybe that's unnecessary…